doc/docs.md

```diff
@@ -2130,6 +2103,69 @@ In this example, the `can_register` method has a receiver of type `User` named `
 The convention is not to use receiver names like `self` or `this`,
 but a short, preferably one letter long, name.
 
+### Embedded structs
+
+V supports embedded structs , similar to inherit, parent-subclasses in other programming languages.
+
+Unlike inherit in other programming languages, Embedded struct in v is just [mixin](https://en.wikipedia.org/wiki/Mixin), where there is no parent-child relationship between embedded structs, and can't type cast.
+
+```v
+struct Color {
```

Comment:
The current example is simpler and easier to understand. I don't see the need to change it.

Reply:
I changed the doc